src/features/account/AccountTokenCurrencyBalance.tsx

```diff
@@ -44,11 +45,14 @@ const AccountTokenCurrencyBalance = ({
 
     switch (ticker) {
       case 'HNT':
-        if (staked) {
-          toCurrencyString(networkStakedBalance).then(setBalanceString)
-        } else {
-          toCurrencyString(networkBalance).then(setBalanceString)
-        }
+        toCurrencyString(
+          networkStakedBalance
```

Comment:
Should this be conditional on `staked` like before? Or ok to use `networkStakedBalance`?

Reply:
I think if you have a `networkStakedBalance` then we should add it no?

Reply:
Yeah that makes sense, not sure what the staked boolean was checking before